public/index.html

```diff
@@ -6,6 +6,43 @@
 		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
 		<title>Homepage</title>
 		<link rel="stylesheet" href="../src/css/index.css" />
+		<link
+			rel="stylesheet"
+			href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+		/>
 	</head>
-	<body></body>
+	<body>
+		<header>
+			<form>
+				<div class="navigation">
+					<div class="navigation__search-wrapper">
+						<input
+							type="search"
+							class="navigation__search-input"
+							placeholder="Search..."
+						/>
+						<button class="navigation__search-button" type="submit">
+							<i class="fa fa-search search-icon fa-lg"></i>
+						</button>
+					</div>
+					<div class="navigation__nav-links">
+						<ul class="navigation__list">
+							<li class="navigation__list--style">
```

Comment:
If you use modifier you need old class plus modifier class
`<li class="navigation__list navigation__list--style"> `this is the right example